scripts/old/circle-build.sh

```diff
@@ -20,6 +20,13 @@ main() {
 }
 
 build() {
```

Comment:
can the scripts/old/* be removed?

Reply:
Yes we could. I chose to kept them for a while as reference in case something comes up. I'd recommend keeping them at least until we confirm the nightlies, snapshots and releases work OK and I didn't miss anything in the refactoring.

Reply:
we can refer to the old git versions if necessary, I'd say let's delete them, perhaps in another PR